07-sampling.Rmd

```diff
@@ -1064,33 +1062,245 @@ Comment on the representativeness of the following *sampling methodologies*:
+This process is called a trial or a Bernoulli trial in honor of Jacob Bernoulli, a 17th-century mathematician who used it already back in those times. 
+Getting a sample of 25 balls is running 25 trials and obtaining 25 numbers, ones or zeros, representing whether we have observed red balls or not on each trial, respectively. 
+The average of these 25 numbers is the sample proportion. 
 
-In Section \@ref(terminology-and-notation), we stated that "statistical inference is the act of making a guess about a population using a sample." But how do we _do_ this inference? In the previous section, we defined the _sampling framework_ only to state that in reality we take one large sample, instead of many samples as done in the sampling framework (which we modeled physically by taking many samples from the bowl). 
+### Random Variables
 
-In reality, we take *only one* sample and use that *one* sample to make statements about the population parameter. This ability of making statements about the population is allowable by a famous theorem, or mathematically proven truth, called the \index{Central Limit Theorem} *Central Limit Theorem*. What you visualized in Figures \@ref(fig:comparing-sampling-distributions) and \@ref(fig:comparing-sampling-distributions-2) and summarized in Tables \@ref(tab:comparing-n) and \@ref(tab:comparing-n-2) was a demonstration of this theorem. It loosely states that when sample means are based on larger and larger sample sizes, the sampling distribution of these sample means becomes both more and more normally shaped and more and more narrow.
+It is useful to represent a trial as a random variable that takes the value of 1 if the ball is red or the value of 0 if the ball is not red. 
+We denote the first trial $X_1$, the second trial $X_2$, and so on. 
+The average of these 25 trials, before they are carried out, can also be expressed as a random variable: $$\bar X = \frac{X_1+X_2+\dots+X_{25}}{25}.$$
+Note that $\bar X$ is a random variable before the trials are run, so $\bar X$ represents all the possible sample proportions that we could obtain. After we run the first trial, if the ball observed is red, we assign $X_1 = 1$, and if it is not red $X_1=0$. Once all 25 trials have been performed, we get a list of 25 ones or zeros. As an illustration, one possible outcome could be: $$\{0,0,0,1,0,1,0,1,0,0,1,0,0,1,1,0,0,0,1,1,0,1,0,0,0,1 \}$$  The observed value of $\bar X$ will be 
+$$\bar X = \frac{0+0+0+1+\dots+0+1}{25} = \frac{10}{25}=0.4.$$ 
+which is both the sample mean of those trial runs and the sample proportion of red balls in the observed sample. 
+Sample proportions are sample means!
```

Comment:
Hmm, maybe this is OK. But I think one additional example walking through another possible outcome before introducing the terminology would be helpful.